x-pack/plugins/triggers_actions_ui/public/index.ts

```diff
@@ -57,6 +57,8 @@ export { enableRule } from './application/lib/rule_api/enable';
 export { disableRule } from './application/lib/rule_api/disable';
 export { muteRule } from './application/lib/rule_api/mute';
 export { unmuteRule } from './application/lib/rule_api/unmute';
+export { snoozeRule } from './application/lib/rule_api/snooze';
```

Comment:
@Zacqary Oops somehow I missed it! Thanks I can now reuse your component. Thanks a ton!